bot.py

```diff
@@ -272,7 +272,8 @@ async def joined(ctx):
 @bot.command(pass_context=True)
 async def roll(ctx, arg1="1", arg2="100"):
     """
-    You can specify the amount of dice with a space or delimited with a 'd', else it will be 2 random nums between 1-6
+    You can specify the amount of dice with a space or delimited with a 'd', \
```

Comment:
With docstrings on multiline comments like so the `\` doesn't need to exist

Reply:
It seems that because it is a help command it prints it out the way that it is. 
![image](https://user-images.githubusercontent.com/18755358/107133214-a2c4f080-689a-11eb-8d6b-a3b6bda85875.png)